lib/shipit.rb

```diff
@@ -173,6 +173,10 @@ def revision
     end
   end
 
+  def feature_bootstrap?
+    secrets.features.include?('bootstrap')
```

Comment:
The tests break here because `features` is `nil`. Might want to do `secrets.features.try!(:include?, 'bootstrap')`.